tutorials/jmsActiveMQ/src/test/java/com/example/jmsActiveMQ/ApplicationTest.java

```diff
@@ -105,9 +111,15 @@ private void startJMSService() throws Exception
     broker.start();
   }
   
+  /**
+   * Create an embedded AMQ broker and a client as the producer for our test.
+   * Create a queue with the supplied queue name.
+   * 
+   * @throws Exception
+   */
   private void createAMQClient() throws Exception 
   {
-    startJMSService();
+    startEmbeddedActiveMQBroker();
     
     // Create a ConnectionFactory
     ActiveMQConnectionFactory connectionFactory = new ActiveMQConnectionFactory("vm://localhost");
```

Comment:
Seems like the argument should come from conf.get("dt.operator.amqIn.prop.connectionFactoryProperties.brokerURL") ?